sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/BeamSqlEnv.java

```diff
@@ -225,13 +225,8 @@ public BeamSqlEnvBuilder autoLoadBuiltinFunctions() {
       return this;
     }
 
-    public BeamSqlEnvBuilder setQueryPlannerClassName(String name) {
```

Comment:
I see that eliminating this method removes a potential source of inconsistency where the options say one planner and the method says another.

Reply:
This would be nice to do, but is impossible as long as 1) we cannot mutate the options and 2) the options are passed down to BeamSqlEnv.

Reply:
Done